css/main.css

```diff
@@ -0,0 +1,1623 @@
+* {
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+body {
+  width: 320px;
+}
+
+nav {
+  width: 100%;
+  height: 80px;
+}
+
+.nav-1 {
+  display: none;
+  margin: 0 0 0 0;
+  padding: 0 0 0 0;
+}
+
+.nav-2 .nav-2child {
+  display: none;
+  width: 100%;
+}
+
+.nav-2 .nav-img {
+  margin-left: 90% !important;
+}
+
+.nav-2 {
+  display: flex;
+  background-color: white;
+  width: 100%;
+  height: 30px;
+  justify-content: space-between;
+  text-align: center;
+  padding-top: 10px;
+  border: 1px solid rgb(226, 224, 224);
+}
+
+.search-icon {
+  width: 20px;
+  height: auto;
+}
+
+.section-1 {
+  display: block;
+  margin-bottom: 100px;
+  height: 475px;
+}
+
+.section-1-1 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/kodak-796x448.jpg");
+  background-size: cover;
+  grid-gap: 2px;
+  height: 200px;
+  width: 90%;
+}
```

Comment:
- [ ] To fix the width in **mobile**, delete `width: 90%;` here
- [ ] To fix vertical alignment in **mobile** in your big images, please delete the height rule